recipes/atool2/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "atool2" %}
+{% set version = "0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/solsticedhiver/atool2/archive/{{ version }}.tar.gz
+  sha256: 87fc55cc21e2ac74c356797fc4e1d73b37dd37a6b91811cea778c68787079239
+
+build:
+  skip: true  # [win] # Missing automake
```

Just to make it easier to read and not confuse with the pre-processor selector.

```suggestion
  # Missing automake
  skip: true  # [win]
```